README.md

```diff
@@ -1,3 +1,35 @@
-# Load-Flexibility-Measures
+# Openstudio Load Flexibility Measures Gem
 
-This is the initial public-facing repository for several OpenStudio measures that use thermal storage for building load flexibility.
+This gem contains measures for thermal energy storage for building cooling and domestic hot water heating.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'openstudio-load-flexibility-measures'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install 'openstudio-load-flexibility-measures'
+
+## Usage
+
+To be filled out later.
+
+## TODO
+
+- [ ] Update measures to code standards
+
+# Releasing
```

Comment:
You may want to add in a few more details regarding the gem release process. 
You can use the [openstudio-ext-gem ](https://github.com/NREL/openstudio-extension-gem) as a model. 
e.g:

# Releasing the gem

* Update CHANGELOG.md
* Run `rake rubocop:auto_correct`
* Update version in `/lib/openstudio/openstudio-load-flexibility-measures/version.rb`
* Create PR to master, after tests and reviews complete, then merge
* Locally - from the master branch, run `rake release`
* On GitHub, go to the releases page and update the latest release tag. Name it “Version x.y.z” and copy the CHANGELOG entry into the description box.